src/components/organisms/SchedulePageModal/SchedulePageModal.tsx

```diff
@@ -25,19 +32,34 @@ export const SchedulePageModal: FC<SchedulePageModalProps> = ({
   isVisible,
 }) => {
   const venueId = useVenueId();
+  useConnectRelatedVenues(venueId);
   const venue = useSelector(currentVenueSelectorData);
   const venueEvents = useSelector(venueEventsSelector);
+  const subvenueEvents = useSelector(subvenueEventsSelector);
+  const parentVenueEvents = useSelector(parentVenueEventsSelector);
+  const siblingVenueEvents = useSelector(siblingVenueEventsSelector);
+
+  const events = useMemo(
+    () =>
+      [
+        ...(venueEvents ?? []),
+        ...(subvenueEvents ?? []),
+        ...(parentVenueEvents ?? []),
+        ...(siblingVenueEvents ?? []),
+      ].sort((a, b) => a.start_utc_seconds - b.start_utc_seconds),
+    [venueEvents, subvenueEvents, parentVenueEvents, siblingVenueEvents]
+  );
```

Comment:
[nit] this could be cleaner by defaulting at the selector

Something like:
```suggestion
  const venue = useSelector(currentVenueSelectorData);
  const venueEvents = useSelector(venueEventsSelector) ?? [];
  const subvenueEvents = useSelector(subvenueEventsSelector) ?? [];
  const parentVenueEvents = useSelector(parentVenueEventsSelector)?? [];
  const siblingVenueEvents = useSelector(siblingVenueEventsSelector)?? [];

  const events = useMemo(
    () =>
      [
        ...venueEvents,
        ...subvenueEvents,
        ...parentVenueEvents,
        ...siblingVenueEvents,
      ].sort((a, b) => a.start_utc_seconds - b.start_utc_seconds),
    [venueEvents, subvenueEvents, parentVenueEvents, siblingVenueEvents]
  );
```